docs/Dreamcast.md

```diff
@@ -8,14 +8,18 @@ _The Sega Dreamcast is a 6th generation home video game console released by Sega
 | :---: | :---: | :---: | :---: | :---: |
 | [Reicast](https://github.com/reicast/reicast-emulator) | dreamcast  | .cdi .chd .gdi | dc_boot.bin, dc_flash.bin | /opt/retropie/configs/dreamcast/mappings |
 | [lr-flycast](https://github.com/libretro/flycast.git) | dreamcast  | .cdi .chd .gdi .zip | dc_boot.bin, dc_flash.bin | /opt/retropie/configs/dreamcast/retroarch.cfg |
+| [Redream](https://redream.io) | dreamcast  | .cdi .chd .gdi | dc_boot.bin, dc_flash.bin | /opt/retropie/configs/dreamcast/mappings |
```

Comment:
_Redream_ supports, `cue/bin` files also. The BIOS files are also different, see https://redream.io/help (see the _Do I need a BIOS?_ section). 
Unsure about mappings, but the `redream`'s config folder is `/opt/retropie/configs/dreamcast/mappings/reicast`.